frontend/src/APIClients/CamperAPIClient.ts

```diff
@@ -2,9 +2,38 @@ import { BEARER_TOKEN } from "../constants/AuthConstants";
 import {
   WaitlistedCamper,
   UpdateWaitlistedStatusType,
+  EditCamperInfoFields,
+  Camper,
 } from "../types/CamperTypes";
 import baseAPIClient from "./BaseAPIClient";
 
+const updateCampersById = async (
+  id: Array<string>,
+  camperData: EditCamperInfoFields,
+): Promise<Array<Camper>> => {
+  try {
+    const body = { camperIds: id, ...camperData };
+
+    // Can't send Map<string, string> over HTTP so need to convert to Object before
+    if (camperData.formResponses) {
+      body.formResponses = Object.assign(
+        {},
+        ...Array.from(camperData.formResponses.entries()).map(([k, v]) => ({
```

Comment:
nit: Instead of k, v you could use something more descriptive like question, response?

Reply:
done